cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -674,12 +678,7 @@ impl pallet_xcm::Config for Runtime {
 	type XcmTeleportFilter = Everything;
 	// Allow reserve based transfer to everywhere except for bridging, here we strictly check what
 	// assets are allowed.
-	type XcmReserveTransferFilter = (
-		LocationWithAssetFilters<WithParentsZeroOrOne, AllAssets>,
-		bridging::to_rococo::AllowedReserveTransferAssets,
-		bridging::to_wococo::AllowedReserveTransferAssets,
-	);
-
+	type XcmReserveTransferFilter = Everything;
```

Comment:
either the comment or config is wrong :)

Reply:
yes, thank you, old comment, comment removed